autoswitch_virtualenv.plugin.zsh

```diff
@@ -25,14 +38,22 @@ function _maybeworkon() {
      workon "$1"
 
      if [ -z "$AUTOSWITCH_SILENT" ]; then
-       # For some reason python --version writes to stderr
-       if type python > /dev/null; then
-           printf "[%s]\n" "$(python --version 2>&1)"
-       elif type python3 > /dev/null; then
-           printf "[%s]\n" "$(python3 --version 2>&1)"
-       else
-           printf "Unable to find python installed on this machine"
-        fi
+        _print_python_version
+     fi
+  fi
+}
+
+
+function _maybepipenv() {
+  if [[ -z "$VIRTUAL_ENV" || "$1" != "$VIRTUAL_ENV" ]]; then
+     if [ -z "$AUTOSWITCH_SILENT" ]; then
+        printf "Switching pipenv: %s  " "$(basename "$1")"
+     fi
+
+     . "$1"/bin/activate
```

Comment:
Just noticed this. How does this work exactly? `.` means current directory but there is a space between it and the activation script path. 

Is this a mistake? Should it be `"$1/bin/activate"`? (`.` is redundant)